SMLHelper/Handlers/CraftDataHandler.cs

```diff
@@ -88,6 +88,18 @@ public static void EditItemSize(TechType techType, Vector2int size)
             CraftDataPatcher.CustomItemSizes[techType] = size;
         }
 
+        /// <summary>
+        /// <para>Allows you to edit item sizes for TechTypes.</para>
+        /// <para>Can be used for existing TechTypes too.</para>
+        /// </summary>
+        /// <param name="techType">The TechType whose item size you want to edit.</param>
+        /// <param name="x">The width of the item</param>
+        /// <param name="y">The hight of the item</param>
+        public static void EditItemSize(TechType techType, int x, int y)
```

Comment:
Very nice. I like this. Very appropriate for a public API.

Reply:
Just to be clear, X is the width and Y is the height, right?

Reply:
Wouldn't make sense otherwise